modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,269 @@
+// CNF-950 4.7 Installing the operators
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
```

Comment:
* scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc

Reply:
Approved.